resources/dicts/patrols/new_cat.json

```
@@ -2633,6 +2633,916 @@
+                "exp": 0,
+                "weight": 20,
+                "art": "gen_cat_looking_up1",
+                "stat_skill": ["DARK,2"],
+                "relationships": [],
+                "new_cat": [
+                    ["dead", "age:senior", "meeting"],
+                    ["dead", "age:senior", "meeting"]
+                ]
+            }
+        ],
+                "antag_success_outcomes": [
+            {
+                "text": "p_l bounds through a thorny bush, a sneer across {PRONOUN/p_l/poss} face. A pair of kittypets meets {PRONOUN/p_l/poss} gaze, startled. As p_l lets out a warning yowl to signify {PRONOUN/p_l/poss} intention to fight, the two kittypets quickly dart away.",
+                "exp": 0,
+                "weight": 20,
+                "art": "gen_angry_cat_warrior",
+                "relationships": [
+                    {
+                        "cats_to": ["n_c:0"],
+                        "cats_from": ["patrol"],
+                        "mutual": true,
+                        "values": ["dislike"],
+                        "amount": 10
+                    },
+                    {
+                        "cats_to": ["n_c:1"],
+                        "cats_from": ["patrol"],
+                        "mutual": true,
+                        "values": ["dislike"],
+                        "amount": 10
+                    }
+                ],
+                "new_cat": [
+                    ["kittypet", "meeting", "existing"],
+                    ["kittypet", "meeting", "existing"]
+                ],
+                "outsider_rep": -2
+            },
+            {
+                "text": "p_l wasn't going to let any cat get the upper paw on {PRONOUN/p_l/object}. Without a second thought, {PRONOUN/p_l/subject} {VERB/p_l/throw/throws} {PRONOUN/p_l/self} at an unsuspecting kittypet, ignoring the scent of sickness on {PRONOUN/n_c:0/object}. Yowls from another kittypet begging p_l to stop makes {PRONOUN/p_l/object} take a step back... but <i>it's too late</i>. The kittypet is dead, p_l has blood on {PRONOUN/p_l/poss} paws and the glare of the other kittypet tells {PRONOUN/p_l/object} that {PRONOUN/p_l/subject} had just made a new enemy.",
+                "exp": 15,
+                "weight": 15,
+                "art": "gen_gen_newcat_hostile",
+                "stat_trait": [
+                "fierce",
+                "bloodthirsty",
+                "cold", 
+                "shameless",
+                "vengeful"
+            ],
+                "relationships": [
+                    {
+                        "cats_to": ["n_c:1"],
+                        "cats_from": ["patrol"],
+                        "mutual": true,
+                        "values": ["dislike"],
+                        "amount": 50
+                    }
+                ],
+                "new_cat": [
+                    ["dead", "meeting", "existing", "kittypet"],
```

```suggestion
                    ["dead", "meeting", "exists", "kittypet"],
```
make sure to fix that mistag in all areas

---

All newcat jsons scanned for it, 3 instances found, fixed